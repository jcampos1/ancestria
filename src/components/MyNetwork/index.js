import React from 'react';
import {saveToken} from 'utils/localStorage/index';
import globalAxios from 'api/index';
import axios from 'axios';

class MyNetwork extends React.Component {
	constructor(props) {
		super(props);
	}

	render() {
		return (
			<h1>
				My Network Component
			</h1>
		)
	}
}

export default MyNetwork;