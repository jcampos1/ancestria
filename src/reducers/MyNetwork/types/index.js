export const SAVE_ACCOUNT_GLOBAL_IN_USE = 'SAVE_ACCOUNT_GLOBAL_IN_USE';