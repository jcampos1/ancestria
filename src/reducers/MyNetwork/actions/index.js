import * as types from 'reducers/MyNetwork/types/index';

export const saveAccountGlobalInUse = accountGlobalInUse => {
	return {
		type: types.SAVE_ACCOUNT_GLOBAL_IN_USE,
		accountGlobalInUse
	}
}