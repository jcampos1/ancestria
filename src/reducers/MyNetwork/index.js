import * as types from 'reducers/MyNetwork/types/index';

const INITIAL_STATE = {
	accountGlobalInUse: null
};

const myNetworkReducer = (state=INITIAL_STATE, action) => {
  switch(action.type) {
  	case types.SAVE_ACCOUNT_GLOBAL_IN_USE:
  		return {
  			...state,
  			accountGlobalInUse: action.accountGlobalInUse
  		};
    default: return state;
  }
}

export default myNetworkReducer;