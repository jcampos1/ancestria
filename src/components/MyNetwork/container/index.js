import { connect } from 'react-redux';
import MyNetwork from 'components/MyNetwork/index';
import {saveAccountGlobalInUse} from 'reducers/MyNetwork/actions/index';

const mapStateToProps = state => ({
	accountGlobalInUse: state.myNetworkReducer.accountGlobalInUse
}); 

const mapDispatchToProps = dispatch => ({
	saveAccountGlobalInUse: accountGlobalInUse => { dispatch(saveAccountGlobalInUse(accountGlobalInUse)) }
});

const MyNetworkContainer = connect(mapStateToProps, mapDispatchToProps)(MyNetwork);
export default MyNetworkContainer;