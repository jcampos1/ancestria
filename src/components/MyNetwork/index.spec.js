import React from 'react';
import {shallow} from 'enzyme';
import MyNetwork from './index';

describe('MyNetwork component', () => {
	it('Should render MyNetwork component', () => {
		const wrapper = shallow(<MyNetwork />);
		expect(wrapper).toMatchSnapshot();
	});
});