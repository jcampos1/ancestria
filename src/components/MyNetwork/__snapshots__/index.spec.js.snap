// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyNetwork component Should render MyNetwork component 1`] = `
<h1>
  My Network Component
</h1>
`;
